documentation/qos_monitor/sdd/qos_monitor_sdd.md

```diff
@@ -34,9 +34,81 @@ The QoS Monitor provides the following services:
 The QoS Monitor consumes the following service:
 * QueryAll private service from the ServiceRegistry core system
 
+In case the QoS Monitor started with the DefaultExternal or the OrchestretedExternal stategy, 
+it consumes the "qos-icmp-ping" service.
+
+The "qos-icmp-ping" service is a composit service consisting of a request-response interaction followed by a sequence of events.
+The "qos-icmp-ping" service provider should provide an endpoint of the "qos-icmp-ping" service.
+The input of the "qos-icmp-ping" service endpoint is an IcmpRequest object.
+
+Input IcmpRequest:
+```json
+{
+    "type": "object",
+    "properties": {
+        "timeToRepeat": {
+            "type": "number"
+        },
+        "timeout": {
+            "type": "number"
+        },
+        "packetSize": {
+            "type": "number"
+        },
+        "ttl": {
+            "type": "number"
+        },
+        "host": {
+            "type": "string"
+        }
+    },
+    "required": [
+                "timeToRepeat",
+                "timeout",
+                "packetSize",
+                "ttl",
+                "host"
+            ]
+}
+```
+
+The response of the "qos-icmp-ping" service endpoint is an IcmpResponseACK object.
+
+Output IcmpResponseACK:
+```json
+{
+    "type": "object",
+    "properties": {
+        "externalMeasurementUuid": {
+            "type": "string",
+            "format": "uuid"
+        },
+        "ackOk": {
+            "type": "string"
+        }
+    },
+    "required": [
+                "externalMeasurementUuid",
+                "ackOk"
+            ]
+}
+```
+
+The response must be followed by the publishing of the following events in the order they listed below:
+
+1. RECEIVED_MONITORING_REQUEST
+1. STARTED_MONITORING_MEASUREMENT
+1. FINISHED_MONITORING_MEASUREMENT
+
+* INTERRUPTED_MONITORING_MEASUREMENT
+
+The "INTERRUPTED_MONITORING_MEASUREMENT" event could precede any other event and it shuld not be followed by any other event.
+
+In order to receive the events invoked by the "qos-icmp-ping" service, the QoS Monitor provide a [notification](https://github.com/arrowhead-f/core-java-spring/blob/qos-monitor-eliminating-icmp4j-dependency/qos-monitor/documentation/QualityOfServiceMonitor-IDD.md#endpoint_post_ping_event_notification) service endpoint.
```

Comment:
This document eventually merged into the ecplise-arrowhead/master branch. Link has to work there

Reply:
changed, please validate

Reply:
done